src/components/mgmt/EventsManagement.vue

```diff
@@ -0,0 +1,289 @@
+          end_time: '2023-12-10 23:00:00',
+        },
+        {
+          title: 'Beach Bonfire Party',
+          capacity: 40,
+          contact_name: 'Emily Davis',
+          contact_email: 'emily1@beachmail.com',
+          contact_phone: '555-234-5678',
+          event_location: 'Golden Beach',
+          descript: 'Join us for a night of fun around the bonfire on the beach.',
+          start_time: '2023-11-15 19:00:00',
+          end_time: '2023-11-15 23:00:00',
+        },
+        {
+          title: 'Poolside BBQ Bash',
+          capacity: 60,
+          contact_name: 'David Wilson',
+          contact_email: 'david1@resortmail.com',
+          contact_phone: '555-345-6789',
+          event_location: 'Main Pool',
+          descript: 'A BBQ party by the pool with great food and music.',
+          start_time: '2023-11-18 14:00:00',
+          end_time: '2023-11-18 18:00:00',
+        },
+        {
+          title: 'Holiday Carnival',
+          capacity: 80,
+          contact_name: 'Sarah Johnson',
+          contact_email: 'sarah1@celebrationmail.com',
+          contact_phone: '555-456-7890',
+          event_location: 'Resort Grounds',
+          descript: 'Get into the holiday spirit with games, rides, and delicious treats.',
+          start_time: '2023-11-23 11:00:00',
+          end_time: '2023-11-23 17:00:00',
+        },
+        {
+          title: 'Festive Luau',
+          capacity: 50,
+          contact_name: 'Michael Smith',
+          contact_email: 'michael2@luau.com',
+          contact_phone: '888-555-1234',
+          event_location: 'Luau Pavilion',
+          descript: 'Celebrate the season with a Hawaiian-themed luau featuring traditional dances and cuisine.',
+          start_time: '2023-12-05 18:30:00',
+          end_time: '2023-12-05 22:30:00',
+        },
+        {
+          title: 'Holiday Movie Night',
+          capacity: 30,
+          contact_name: 'Alice Johnson',
+          contact_email: 'alice2@holidaycinema.com',
+          contact_phone: '555-987-6543',
+          event_location: 'Resort Cinema',
+          descript: 'Enjoy classic holiday films under the stars with popcorn and hot cocoa.',
+          start_time: '2023-12-20 20:00:00',
+          end_time: '2023-12-20 23:00:00',
+        },
+        {
+          title: 'Snowy Ski Trip',
+          capacity: 25,
+          contact_name: 'John Fritter',
+          contact_email: 'john1@snowyski.com',
+          contact_phone: '555-234-5678',
+          event_location: 'Mountain Lodge',
+          descript: 'Hit the slopes and enjoy a snowy adventure with skiing and cozy cabins.',
+          start_time: '2023-12-28 09:00:00',
+          end_time: '2023-12-29 16:00:00',
+        }
+      ],
+      eventsCols: [
+        {
+          id: 'title',
+          label: 'Title',
+        },
+        {
+          id: 'capacity',
+          label: 'Capacity',
+        },
+        {
+          id: 'contact_name',

```

Comment:
Elsewhere in the project (and even in this file), we are using hyphens for compound names. You can use either, but let's stick with one convention for the project. Could you change the underscores to hyphens in eventsCols and eventsRows?